financial-connections/src/test/java/com/stripe/android/financialconnections/features/institutionpicker/InstitutionPickerViewModelTest.kt

```diff
@@ -35,9 +42,11 @@ internal class InstitutionPickerViewModelTest {
 
     private val searchInstitutions = mock<SearchInstitutions>()
     private val featuredInstitutions = mock<FeaturedInstitutions>()
-    private val getManifest = mock<GetManifest>()
+    private val sync = mock<GetOrFetchSync>()
```

Comment:
I’d love for us to use fakes instead of mocks. That would make encoding behavior easier and reduces our dependency on Mockito. Fine to keep as is for now, but might be a good testability improvement down the line.